identifiers/country-pt.csv

```diff
@@ -1,24 +1,44 @@
-ocd-division/country:pt/region:vr,Vila Real
+id,name,sameAs,sameAsNote
+ocd-division/country:pt,Portugal,,
+ocd-division/country:pt/autonomous_region:ac,Região Autónoma dos Açores,,
+ocd-division/country:pt/autonomous_region:ma,Região Autónoma da Madeira,,
+ocd-division/country:pt/cd:europa,Europa,,
+ocd-division/country:pt/cd:fora_da_europa,Fora da Europa,,
+ocd-division/country:pt/district:av,Aveiro,ocd-division/country:pt/region:av,Portugal is divided into eighteen distritos
+ocd-division/country:pt/district:ba,Bragança,ocd-division/country:pt/region:ba,Portugal is divided into eighteen distritos
+ocd-division/country:pt/district:be,Beja,ocd-division/country:pt/region:be,Portugal is divided into eighteen distritos
+ocd-division/country:pt/district:br,Braga,ocd-division/country:pt/region:br,Portugal is divided into eighteen distritos
+ocd-division/country:pt/district:cb,Castelo Branco,ocd-division/country:pt/region:cb,Portugal is divided into eighteen distritos
+ocd-division/country:pt/district:co,Coimbra,ocd-division/country:pt/region:co,Portugal is divided into eighteen distritos
+ocd-division/country:pt/district:ev,Évora,ocd-division/country:pt/region:ev,Portugal is divided into eighteen distritos
+ocd-division/country:pt/district:fa,Faro,ocd-division/country:pt/region:fa,Portugal is divided into eighteen distritos
+ocd-division/country:pt/district:gu,Guarda,ocd-division/country:pt/region:gu,Portugal is divided into eighteen distritos
+ocd-division/country:pt/district:le,Leiria,ocd-division/country:pt/region:le,Portugal is divided into eighteen distritos
+ocd-division/country:pt/district:li,Lisboa,ocd-division/country:pt/region:li,Portugal is divided into eighteen distritos
+ocd-division/country:pt/district:pa,Portalegre,ocd-division/country:pt/region:pa,Portugal is divided into eighteen distritos
+ocd-division/country:pt/district:po,Porto,ocd-division/country:pt/region:po,Portugal is divided into eighteen distritos
+ocd-division/country:pt/district:sa,Santarém,ocd-division/country:pt/region:sa,Portugal is divided into eighteen distritos
+ocd-division/country:pt/district:se,Setúbal,ocd-division/country:pt/region:se,Portugal is divided into eighteen distritos
+ocd-division/country:pt/district:vc,Viana do Castelo,ocd-division/country:pt/region:vc,Portugal is divided into eighteen distritos
+ocd-division/country:pt/district:vi,Viseu,ocd-division/country:pt/region:vi,Portugal is divided into eighteen distritos
+ocd-division/country:pt/district:vr,Vila Real,ocd-division/country:pt/region:vr,Portugal is divided into eighteen distritos
+ocd-division/country:pt/region:av,Aveiro,,
+ocd-division/country:pt/region:ba,Bragança,,
+ocd-division/country:pt/region:be,Beja,,
+ocd-division/country:pt/region:br,Braga,,
+ocd-division/country:pt/region:cb,Castelo Branco,,
+ocd-division/country:pt/region:co,Coimbra,,
+ocd-division/country:pt/region:europa,Europa,ocd-division/country:pt/cd:europa,Because Europa is more region as congressionals

```

Comment:
Could you explain this one a bit more? I don't understand which entity this is.

Reply:
Hello, 

As discussed before, I add aliases to add country-specific types. 
So, the standard type for europa is cd. But as @sguenther85 we can use region instead.

Reply:
Please check : https://en.wikipedia.org/wiki/2019_Portuguese_legislative_election. Under the "Electoral system" section, you can find : 

> Each one of Portugal's eighteen administrative districts, as well as each one of the country's two autonomous regions - the Azores and Madeira - is an electoral constituency. Portuguese voters residing outside the national territory are grouped into two electoral constituencies - Europe and the rest of the world - each one of which elects two Assembly members.

Reply:
Understood.

This is a tricky one since, technically, the entirety of Europe is not contained within the borders of Poland, which is what this OCD-ID implies. :)

In this case I would recommend using something like
`ocd-division/country:pt/cd:europa`
and
`ocd-division/country:pt/cd:global`
to indicate that these are constituency districts (or swap out `cd` for something else to indicate a legislative district).

Reply:
If we use global, it may refer to europe and outside of europe. So, I can use ocd-division/country:pt/cd:outside_europe. 
What do you think ?

Reply:
Even

`ocd-division/country:pt/cd:europa`
`ocd-division/country:pt/cd:fora_da_europa`

would convey the right information, IMO. Thanks!

Reply:
If I understand you wanna remove the aliases. Because, I already defined the following lines OCD IDs before :
ocd-division/country:pt/cd:europa,Europa,,
ocd-division/country:pt/cd:fora_da_europa,Fora da Europa,,

Do you suggest to remove the aliases ?